deny.toml

```diff
@@ -35,16 +36,8 @@ targets = [
 db-path = "~/.cargo/advisory-db"
 # The url of the advisory database to use
 db-urls = ["https://github.com/rustsec/advisory-db"]
-# The lint level for security vulnerabilities
-vulnerability = "deny"
-# The lint level for unmaintained crates
-unmaintained = "warn"
 # The lint level for crates that have been yanked from their source registry
 yanked = "warn"
-# The lint level for crates with security notices. Note that as of
-# 2019-12-17 there are no security notice advisories in
-# https://github.com/rustsec/advisory-db
-notice = "warn"
```

Comment:
Hmmm, I am not very familiar with this logic, but I think we should keep these.

Reply:
@serban300 me neither, but the actual documentation: https://embarkstudios.github.io/cargo-deny/checks/advisories/cfg.html#the-version-field-optional says:
```
vulnerability - Removed, all vulnerability advisories now emit errors.
unmaintained - Removed, all unmaintained advisories now emit errors.
notice - Removed, all notice advisories now emit errors.
```
iiuc, it should be even more strict now and ok

Reply:
Oh, ok. Sounds good then ! Thanks !